mne/beamformer/_dics.py

```diff
@@ -676,6 +676,10 @@ def tf_dics(epochs, forward, noise_csds, tmin, tmax, tstep, win_lengths,
         raise ValueError('When using multitaper mode and specifying '
                          'multitaper transform bandwidth, one value must be '
                          'provided per frequency bin')
+    if isinstance(cwt_n_cycles, int) or isinstance(cwt_n_cycles, float):
```

Comment:
```suggestion
    if isinstance(cwt_n_cycles, [int, float]):
```

Reply:
I thought that `isinstance` required that its second argument be a tuple (not a list)?